README.md

```diff
@@ -97,25 +99,37 @@ See the basic usage syntax below (you can check it out with `summon --help` comm
 
 ```
 summon PROJECT_NAME [with [OPTIONS]] [without [OPTIONS]]
+       [-f|--file FILENAME] [--package CUSTOM_PRELUDE_PACKAGE]
+       [--module CUSTOM_PRELUDE_MODULE]
 
 Available global options:
   -h, --help               Show this help text
   -f,--file FILENAME       Path to the toml file with configurations. If not
                            specified '~/summoner.toml' will be used if present
+  --package CUSTOM_PRELUDE_PACKAGE
```

Comment:
Maybe this option should have name `--prelude-package` in CLI? Because `--package` option names is ambiguous. And you can set `metavar` name to just `PACKAGE_NAME`.